### ISSUE TITLE
Project Checkpoint Four

### ISSUE THREAD

**Issue by frabbitry** [robftg/Rocket-Launches#19]
@biancamaginley @robftg 

Hey guys. I apologize for the fact that this review is a little bit later than it should be. I'll admit that I kind of forgot about writing them up, with all of this quarantine craziness. 

I know that you guys blazed through your markup and got it finished well ahead of the deadline. There were a few small changes that I think you guys wanted to make that haven't been done yet, such as standardizing spellings for things like the Kennedy Space Center, but I think that, markup-wise, you guys are in a pretty good place.

This is the point in the semester when we really want to start focusing on the data visualization aspect of the project, and so that's what I'm going to recommend that you guys start working on. I know that you both are busy, but I think that your project has several options for data visualization moving forward, none of which are going to require massive amounts of effort.

Option One: SVG 
You guys are going to learn more about SVG in the upcoming unit, but there's some pretty obvious applications of SVG for your project. I think that Dr. B is going to be exploring making an SVG timeline from your project data, already, as a class example. You guys could do something like that, using the time data you've already got encoded. 

Option Two: Network Graphs
I also think that you guys already have a lot already in your markup that you could use to generate interesting network graphs. You could look at how things like launch base are related to mission duration, or how inclination is related to altitude. All you need to do is start writing XQuery scripts to generate a CSV or TSV. I know that @biancamaginley is having trouble getting access to Cytoscape, so perhaps a  fair division of labor would be for Bianca to generate a CSV or TSV with Exide and @robftg to import that into Cytoscape and explore some styles for the network graph. 

Option Three: Mapping
I think that I mentioned mapping as an interesting possibility for your project, but it also might be the most labor-intensive. To do this, you'd want to go in and insert geocoordinates, your lats and lons, for your launch bases and your landing sites. You might be able to do that with an API, which would automate part of that process. It's still an extra step, though. That being said, I think that this might be a really interesting route to take, because then you could do things like analyze how historical weather patterns influenced landing sites and such. One advantage that you guys definitely have is that you're working with dates in the 20th and 21st centuries, which means that there is historical data for that time period out there and it's fairly accurate. Especially for weather patterns. Then again, you'd also need to take an extra step of finding weather geodata and working with QGIS to make that a layer on your map. Of course, I'd be willing to help you guys find stuff and give you some advice on how to turn it into a layer, if you choose to explore this particular data visualization option.

Once you guys start coming up with some different visualizations of your project data, then you can start really filling out the website with pictures of these visualizations and explanations of the visualizations. As it is now, the website could use some base-level work, such as a more thorough explanation of what the project is about and who's on the team. 

In short, you guys have just about finished the markup aspect of the course. Time to start working on data visualization!
